docs/concepts/capacity-credits-concept.md

```diff
@@ -29,7 +29,7 @@ Every time you authenticate with Lit, the request context (i.e. wallet address,
 
 A `Capacity Credits NFT` can be very easily minted from the Lit Explorer. So, here we will show how you can mint it using `contracts-sdk`. You can download the `contracts-sdk` from `npm` [here](https://www.npmjs.com/package/@lit-protocol/contracts-sdk).
 
-You’ll also need some 'testLPX' tokens for minting. These are test tokens that hold no real value and should only be used to pay for usage on Habanero. `testLPX` should only be claimed from the verified faucet, linked [here](https://faucet.litprotocol.com/).
+You’ll also need some 'tstLPX' tokens for minting. These are test tokens that hold no real value and should only be used to pay for usage on Datil-prod. `tstLPX` should only be claimed from the verified faucet, linked [here](https://faucet.litprotocol.com/).
```

Comment:
maybe we should just update the URL to direct to the new one though

Reply:
well, since we've told some select folks to continue using Habanero, maybe unneccessary right now. The call out box is enough i think. But we should ensure this link directs to the Yellowstone faucet

Reply:
Updated all of the faucet links to Yellowstone